src/main/java/duke/task/TaskType.java

```diff
@@ -0,0 +1,40 @@
+package duke.task;
+
+import duke.exception.DukeException;
+
+/**
+ * Represents the type of task.
+ */
+public enum TaskType {
+
+    DEADLINE("D"),
```

Comment:
I'm not 100% sure on this, but I think the coding standard mention that enums should be in pascal case, not capital case